src/methods/sep8Send/revisePaymentTransaction.ts

```diff
@@ -52,10 +52,35 @@ export const revisePaymentTransaction = async ({
   // parse SEP-8 response
   const sep8ApprovalResultJson = await sep8ApprovalResult.json();
   switch (sep8ApprovalResultJson.status) {
+    case Sep8ApprovalStatus.ACTION_REQUIRED:
+      log.response({
+        title: "Action Required",
+        body: "Additional information is needed before we can proceed",
```

Comment:
nit: punctuation 
```suggestion
        body: "Additional information is needed before we can proceed.",
```

Reply:
Done in https://github.com/stellar/stellar-demo-wallet/pull/164/commits/b4f2fb2d3764d20150c4946727335e4ed1f1b625